### DISCUSSION TITLE
Question concerning Codeception Tests

### DISCUSSION THREAD

**Discussion by astridx** [joomla/joomla-cms#32811] [Q&A]
When I run all the tests, I get an error. I have attached the HMTL-log (renamed as .txt) and my system informations here. The error occurs when testing the renaming of a folder in the Media Manager. In my case, the folder is not created before renaming. If I add the creation of the folder before the test, then all tests run without errors.

I am wondering why this error does not occur here in the Git environment. Can someone tell me what is different for me than here in the git environment?

[systeminfo-2021-03-22T19 19 09+00 00.txt](https://github.com/joomla/joomla-cms/files/6184936/systeminfo-2021-03-22T19.19.09%2B00.00.txt)

[report.txt](https://github.com/joomla/joomla-cms/files/6184946/report.txt)

**Comment by dgrammatiko** [comment]
Try adding some delay so the system has enough time to complete the task before trying to ask if it's done

**Reply by astridx** [reply]
Unfortunately, the delay does not help either
